src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -294,6 +320,8 @@ def _export_data(self, data, thumb_nail):
         """
         filename = self._filename.value
         if data:
+            self._add_feature_info_filename()
+            filename = self._filename.value
```

Comment:
I think it'd best so that it doesn't change the VA:
```suggestion
            filename = self._add_feature_info_filename(filename)
```